src/Symfony/Component/Validator/Constraints/ExpressionLanguageSyntax.php

```diff
@@ -31,14 +31,20 @@ class ExpressionLanguageSyntax extends Constraint
     public $message = 'This value should be a valid expression.';
     public $service;
     public $allowedVariables;
+    public $allowNullAndEmptyString = false;
 
-    public function __construct(array $options = null, string $message = null, string $service = null, array $allowedVariables = null, array $groups = null, $payload = null)
+    public function __construct(array $options = null, string $message = null, string $service = null, array $allowedVariables = null, bool $allowNullAndEmptyString = null, array $groups = null, $payload = null)
```

Comment:
Shouldn't we move the parameter at the end for BC? (I wonder if it matters for annotations :thinking:)